test/data/testinit.js

```diff
@@ -276,6 +276,7 @@ this.loadTests = function() {
 				"unit/basic.js",
 
 				"unit/core.js",
+				"unit/readywait.js",
```

Comment:
Adding a new module means more than extending this list. Can you instead put the new test in test/unit/ready.js?